packages/design-tokens/src/tokens/color/background.ts

```diff
@@ -11,10 +11,10 @@ export const background = {
 	danger: colorPaletteRed50,
 	primary: colorPaletteWhite,
 	secondary: colorPaletteGrey100,
-	sideNav: '#021c6b',
+	'side-nav': '#021c6b',
```

Comment:
why the dash? I thought we supported camel casing?

Reply:
stitches needs these to be `kebab` cased, moved this down to the theme level instead of adjusting the tokens.